### ISSUE TITLE
Implement new homepage design

### ISSUE THREAD

**Issue by Dahianna** [18F/18f.gsa.gov#3207]
- [x] Review design with leadership
- [x] Make microrequest for a designer to make edits and implement

Here are the mockups on InVision:
https://gsa.invisionapp.com/share/PMUWVRMSUKN

Sketch files for USWDS 2.0 are here on Drive in the site folder:
https://drive.google.com/open?id=1jGVEhjizHt7ZZ0eFmqedlccs1O6y69Vh

Photos from 18F workshops here on Drive:
https://drive.google.com/open?id=1ySWACkfQXw3n9yA1-KotGrBKN9Ue_I-T

**Comment by Dahianna** [comment]
Here are the results for the survey. As we've seen with our internal interviews, desktop 4 seems to be the preference. It's not perfect and we have to work on updating the content, but the illustration was seen more favorable over the hero image https://app.mural.co/invitation/mural/gsa6/1577730854809?sender=dahiannasalazarforeman2861&key=06243d9e-8f83-494b-894c-33787fd1bd81

**Comment by Dahianna** [comment]
More about the research https://github.com/18F/18f.gsa.gov/issues/3061#issuecomment-554079382

**Comment by lauraponce** [comment]
@Dahianna here's the updated version of the homepage: [Invision link](https://gsa.invisionapp.com/share/38ZS7VYMNZ2#/441027931_Desktop_4-Updated) 

The updates include:
- changing "agency" to "consultancy" in the headline
- changing "featured projects" to "case studies"

The content on version 3 differs from 4 where it says "We support your mission" in 3 vs "We help you achieve your mission" in 4. Let me know if you prefer the version 3 phrasing and I can update that. 

Anne also mentioned adding 'Guides' to the top nav, let me know if you'd like that in this mockup.

![Desktop 4-updated](https://user-images.githubusercontent.com/21222704/102823175-43b12800-438f-11eb-82b3-3b1512f3348f.png)

**Comment by Dahianna** [comment]
Thanks for the quick turnaround @lauraponce ! So far so good. Yes, let's change "We help you achieve your mission" to "We support your mission" 

Let's add guides as well. And I was thinking we should update the nav order to:
`Projects - Our Processes - Guides - About us - Blog`
What do you think?

**Comment by lauraponce** [comment]
@Dahianna I agree with that ordering for the nav — updated!

Here's the new version with the "we support your mission" language and the updated nav order: 
[Invision link](https://gsa.invisionapp.com/share/38ZS7VYMNZ2#/441030052_Desktop_4-Updated_New) and the [updated Sketch file](https://drive.google.com/drive/folders/1jGVEhjizHt7ZZ0eFmqedlccs1O6y69Vh).

![Desktop 4-updated new](https://user-images.githubusercontent.com/21222704/102825344-6ba28a80-4393-11eb-8f9a-e9b9219e2b9c.png)

**Comment by Dahianna** [comment]
Awesome, thanks @lauraponce !

**Comment by Dahianna** [comment]
@apburnes this is ready to implement. Let me know if we need to submit a #helpwanted request

**Comment by apburnes** [comment]
@Dahianna I'll be fairly busy this week so maybe we throw this into the `helpwanted` tag. Also, we may want to break this up into a few issues to make it easier on the folks jumping in to contribute.

**Comment by Dahianna** [comment]
Sounds good, I'll go ahead and do that. Thanks!

**Comment by cmajel** [comment]
@Dahianna I've updated [the drive](https://drive.google.com/drive/folders/1jGVEhjizHt7ZZ0eFmqedlccs1O6y69Vh) with an updated header graphic and sketch file that incorporates more 18F brand colors. Before and after below. 

I'd also recommend we use [$color-dark for headings](https://18f.gsa.gov/styleguide/colors/), rather than black, to make the new homepage consistent with other pages. I've included an example below, and it's also in my updated sketch file.

If these changes seem right, I can update the Invision link. 

## hero graphic

### original
![homepage-graphic--original](https://user-images.githubusercontent.com/2374206/103792280-71de6b00-5011-11eb-8845-109e1bfcfff1.png)

### updated
![homepage-graphic--recolor](https://user-images.githubusercontent.com/2374206/103792302-799e0f80-5011-11eb-8629-a810753def97.png)

## heading colors

### original
![homepage-headings--original](https://user-images.githubusercontent.com/2374206/103793493-08ab2780-5012-11eb-9402-268f176f6a5b.png)

### updated
![homepage-headings--recolor](https://user-images.githubusercontent.com/2374206/103793545-0c3eae80-5012-11eb-97b9-7895a65a7ce9.png)

**Comment by Dahianna** [comment]
@lauraponce the site team also noticed the new design doesn't include the search bar. With the "Gen in touch" button now also on the nav, where could we place the search bar?

**Comment by Dahianna** [comment]
@cmajel Thank you for the updates. They look good to me

**Comment by lauraponce** [comment]
@Dahianna here's what it looks like with the search bar on the right side of the nav, it seems like it can still fit in there. Thoughts on this?

![Screen Shot 2021-01-06 at 10 27 11 AM](https://user-images.githubusercontent.com/21222704/103806376-c7167e80-5009-11eb-8d5d-6bab8565f646.png)

**Comment by Dahianna** [comment]
Thanks @lauraponce What do you think about moving the get in touch button after the search bar? 
cc @apburnes thoughts?

**Comment by lauraponce** [comment]
@Dahianna here's what it looks like with that swap, I feel like it works pretty well like this: 

![Screen Shot 2021-01-06 at 12 07 39 PM](https://user-images.githubusercontent.com/21222704/103815185-d3a1d380-5017-11eb-8eda-5f266b5701f3.png)

**Comment by apburnes** [comment]
The header container is currently using the USWDS standard at 944px from edge to edge for all of the elements in the header.  What is the mock container width from the edge of the logo to the edge of the `Get in touch` button?

**Comment by lauraponce** [comment]
It should be the same as the USWDS standard, this is 944px

![Screen Shot 2021-01-06 at 12 42 49 PM](https://user-images.githubusercontent.com/21222704/103818310-1dd98380-501d-11eb-857b-ac49343766a0.png)

**Comment by Dahianna** [comment]
Great! Thank you so much @lauraponce ! I'll share with @AshleyO85 and the leadership team to see if they have any last comments, other than that we'll be ready to do the update!

**Comment by Dahianna** [comment]
@lauraponce when you have a moment can you update the invision with the changes to the header and Christine's update to the hero. Thanks!

**Comment by AshleyO85** [comment]
is this different the web page revision i owe you in staffing


18F - U.S. General Services Administration

*Ashley T. Owens*

Acting Deputy Director and Senior Advisor of Acquisition (Acq Breaker of
Chains)

GSA - FAS - TTS - 18F

*Build Better, Buy Better*

**Comment by Dahianna** [comment]
Yep. This design and content had been approved by leadership in the past. I
just want to double check that it resonates with you too and you agree with
the new names for the sections in the site.

U.S. General Services Administration

*Dahianna V. Salazar Foreman*
 Digital Marketing
 TTS Outreach
 Technology Transformation Services
 Federal Acquisition Service
  <https://www.linkedin.com/company/gsa-tts/>  <https://twitter.com/GSA_TTS>

**Comment by AshleyO85** [comment]
can we add a quote from DOJ CRT and update the case studies or is that out of scope for this issue?

**Comment by Dahianna** [comment]
If you give me the quote we can update that. As for the case studies, that's a different issue. @anjunainaustin is working on that.

**Comment by lauraponce** [comment]
Hi @Dahianna! @lauraGgit @cmajel and I were chatting over Slack about this and I wanted to add the designs here also for visibility: here's the latest HP design and it's updated in the [Drive folder](https://drive.google.com/drive/folders/1jGVEhjizHt7ZZ0eFmqedlccs1O6y69Vh) also (marked as LATEST with the timestamp). 

![Latest HP design](https://user-images.githubusercontent.com/21222704/107091897-5c4a9580-67b7-11eb-8ffe-113a45fe4d17.png)

**Comment by Dahianna** [comment]
Thank you @lauraponce @lauraGgit and @cmajel ! I only have two minor edits. @AshleyO85 wants to change the FEC quote to a DOJ quote. I'm waiting on the content. Also let's make sure we use the [current footer and identifier](https://18f.gsa.gov/) @lauraGgit feel free to start the pull request.
